guide/sending-single-delivery/sending-my-first-order.md

```diff
@@ -0,0 +1,148 @@
+# Option B: Sending a Single Delivery per order line
+
+This page explains how to request deliveries for order lines by means of a Single Delivery per order line.  
+Order lines with the same `originalPosition` are grouped by Tradecloud One, meaning that their (single) deliveries are merged into a delivery schedule in one order line.
```

Comment:
- using the `originalPosition` is a new option for the reader, explain when to use this option. I even think this should be postponed, because it not relevant at this point, but when either the buyer splits or the supplier splits (both may have their own guides).
- remove "(single)" brackets

Reply:
Removed, let's keep it simple indeed.